app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -611,12 +611,16 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       });
   };
 
-  $scope.updateAuthStatus = function(updatedValue) {
```

Comment:
don't we need the `$scope.updateAuthStatus` function any more ? 
It is removed in a PR that does not deal with authStatus, what am I missing ?

Reply:
@moolitayer I think you removed the wrong function.
You probably meant to remove `updateAlertsHostname`.

@yaacov Thanks.

Reply:
fixed, thanks